manifests/s/Synology/SurveillanceStationClient/2.2.1-2564/Synology.SurveillanceStationClient.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Synology.SurveillanceStationClient
+PackageVersion: 2.2.1-2564
+PackageLocale: en-US
+Publisher: Synology
+PublisherUrl: https://www.synology.com
+PublisherSupportUrl: https://www.synology.com/company/contact_us
+PrivacyUrl: https://www.synology.com/privacy
+PackageName: Surveillance Station Client
+License: Copyright (c) Synology Inc.

```

```suggestion
License: Proprietary
Copyright: Copyright (c) Synology Inc.
```

A copyright isn't a license